README.md

```diff
@@ -36,15 +36,29 @@ build the container from source
 docker build -t rest_api:local .
 ```
 
-run the container. Please use the correct link for the hostname `elastic`, either with the `--link` or the `--ad-host` option
+and then run the same container. 
+Notice you can specify the elasticsearch server using the ELASTICSEARCH_URL environment variable:
 ```bash
-docker run -d -p 8008:80 --name cttv_rest_api --add-host elastic:10.0.2.2 --ulimit nofile=65535:65535 -e "CTTV_API_CONFIG=dockerlink" rest_api:local
+docker run -d -p 8008:80 --name opentargets_rest_api -e "ELASTICSEARCH_URL=http://localhost:9200" --privileged rest_api:local
 ```
 
-or get it from [quay.io](https://quay.io/repository/cttv/rest_api)
+or get it from [docker hub](https://hub.docker.com/r/opentargets/rest_api/builds/)
 ```bash
-docker run -d -p 8008:80 --name cttv_rest_api --add-host elastic:10.0.2.2 --ulimit nofile=65535:65535 -e "CTTV_API_CONFIG=dockerlink" quay.io/cttv/rest_api
+docker run -d -p 8008:8008 --name opentargets_rest_api -e "ELASTICSEARCH_URL=http://localhost:9200" --privileged opentargets/rest_api
 
 ```
+Notice that if you try to map port 80 inside the container with `-p 8008:80` you may get a `403 access forbidden` as it will check the domain to be `*.targetvalidation.org`.

```

Comment:
this is true only for the webapp connecting to the api, it will use for normal calls